### ISSUE TITLE
Bootstrap using wrong JS version compared to CSS versions

### ISSUE THREAD

**Issue by hsankala** [cloudscribe/cloudscribe.templates#45]
Hey up Joe. Using latest VS template 1.73.   In this template we're using the latest bootstrap version 4.3.1, the CSS is getting built by the gulp task.

The Bootstrap JS is at the wrong version v4.1.2, in these places in the template:

- Root wwwroot bootstrap [bootstrap.js](https://github.com/cloudscribe/cloudscribe.templates/blob/master/Content/WebApp/wwwroot/js/bootstrap.js)
- Site bootstrap bundles [bootstrap.bundle.js](https://github.com/cloudscribe/cloudscribe.templates/blob/master/Content/WebApp/sitefiles/s1/themes/custom1/wwwroot/js/bootstrap.bundle.js)
- The all theme bootstrap JS bundles e.g [Lux theme bootstrap.bundle.js](https://github.com/cloudscribe/cloudscribe.templates/blob/master/Content/WebApp/SharedThemes/lux/wwwroot/js/bootstrap4.bundle.js)

Might be worth having a gulp task to copy these from the nodemodules folder?

**Comment by hsankala** [comment]
Noticed that our [_bs4.scss](https://github.com/cloudscribe/cloudscribe.templates/blob/master/Content/WebApp/app-scss/_bs4.scss) is also missing two entries:

@import "node_modules/bootstrap/scss/spinners";
@import "node_modules/bootstrap/scss/toasts";

**Comment by joeaudette** [comment]
I've just published v1.75 all themes have been updated to latest bootstrap. Note that this release also changes from netcoreapp2.1 to netcoreapp2.2 with in process hosting enabled.

netcoreapp2.2 requires the .NET Core 2.2 SDK which is included with VS 2019 assuming web workloads installed. VS 2017 ships with .NET Core 2.1 SDK so the 2.2 SDK has to be installed separately for VS 2017 if it is not already installed.
